iothub_client/src/iothub_client_ll_uploadtoblob.c

```diff
@@ -42,8 +42,8 @@ int snprintf(char * s, size_t n, const char * format, ...)
 #endif
 
 /*Codes_SRS_IOTHUBCLIENT_LL_02_085: [ IoTHubClient_LL_UploadToBlob shall use the same authorization as step 1. to prepare and perform a HTTP request with the following parameters: ]*/
-#define FILE_UPLOAD_FAILED_BODY "{ \"isSuccess\":false, \"statusCode\":-1,\"statusDescription\" : \"client not able to connect with the server\" }"
-#define FILE_UPLOAD_ABORTED_BODY "{ \"isSuccess\":false, \"statusCode\":-1,\"statusDescription\" : \"file upload aborted\" }"
+#define FILE_UPLOAD_FAILED_BODY "{ \"correlationId\":\"%s\", \"isSuccess\":false, \"statusCode\":-1,\"statusDescription\" : \"client not able to connect with the server\" }"
```

Comment:
I don't think we need this anymore since we have other places where the formatting string is used.
We should instead use the same formatting string and the arguments separately (i.e. "false"|"true", `statusDescription`s, etc).

Reply:
good idea thanks for that. pushed a change to simplify this.